src/main/java/iotask/Main.java

```diff
@@ -0,0 +1,15 @@
+package iotask;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
```

Comment:
its better to handle exception directly in method which throws it

Reply:
fixed